keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -15,33 +15,27 @@
 
 import tensorflow as tf
 
-from keras_cv.utils import preprocessing
 from keras_cv.layers import BaseImageAugmentationLayer
 
 
 @tf.keras.utils.register_keras_serializable(package="keras_cv")
 class RandomResizedCrop(BaseImageAugmentationLayer):
-    """Randomly resizes an image then crops to a target size. 
-	Code credits: https://github.com/keras-team/keras-cv/pull/457
+    """
+    Randomly crops a part of an image and resizes it to provided size.
+
     Args:
-        resize_factor: A tuple of two floats, a single float or `keras_cv.FactorSampler`.
-            `factor` controls the extent to which the image is resized.
-            `factor=0.0` makes this layer perform a no-op operation, while a value of
-            1.0 doubles the size of the image.  If a single float is used, a value
-            between `0.0` and the passed float is sampled.  In order to ensure the value
-            is always the same, please pass a tuple with two identical floats:
-            `(0.5, 0.5)`.
-        crop_size: a tuple of two integers used as the target size to crop images to.
+        target_size: A uple of two integers used as the target size to crop 
+            images to.
+        aspect_ratio_factor: (Optional) A tuple of two floats. Represents the 
```

Comment:
I wonder if there is a such thing as a "sensible default" here.  Does torchvision use 3/4 and 4/3, or was that a number we picked?  Unsure on this as of now.

Reply:
@LukeWood 

Yes torchvision uses the same. I believe these defaults are also used in the original paper, but I'm not sure.

Reply:
Which paper are you referring to @AdityaKane2001? Inception?

Reply:
My bad. The original paper did not have any defaults. I recollected this snippet from SimCLR:
![image](https://user-images.githubusercontent.com/64411306/174793226-24c00d72-7da7-4184-ad83-305bdd2c6b70.png)

Reply:
yeah, I don't think we want any defaults.